pkg/validations/cluster.go

```diff
@@ -9,9 +9,30 @@ import (
 
 func ValidateTaintsSupport(clusterSpec *cluster.Spec) error {
 	if !features.IsActive(features.TaintsSupport()) {
-		if len(clusterSpec.Cluster.Spec.ControlPlaneConfiguration.Taints) > 0 {
+		wngcTaintsPresent := false
```

Comment:
- [ ] Ticket to come back and cleanup these `wngc` names

Reply:
- [ ] test